src/node_crypto.h

```diff
@@ -488,22 +488,21 @@ class Hmac : public BaseObject {
 
   static void Initialize(Environment* env, v8::Local<v8::Object> target);
 
- protected:
```

Comment:
This just seems to turn a couple of `protected` properties into `public` ones, which would be a `semver-minor` change (if this is a public API). But this header is one of those with a `NODE_WANT_INTERNALS`, so I don’t think we even consider this public.